src/wp-admin/update-core.php

```diff
@@ -548,7 +548,7 @@ function do_core_upgrade( $reinstall = false ) {
 				show_message( sprintf(
 					/* translators: URL to support forum */
 					__( 'If you see this message after waiting 15 minutes and trying the update again, please make a post on our <a href="%s">support forum</a>.' ),
-					'https://forums.classicpress.net/c/support/'
+					'https://forums.classicpress.net/c/supportc/support/'
```

Comment:
One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.